src/App/App.jsx

```diff
@@ -1,16 +1,21 @@
 import React from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
 import firebase from "firebase/app";
+import Login from "../Login/Login";
+import Home from "../Home/Home";
+import { AuthProvider } from "../Auth";
+import PrivateRoute from "../PrivateRoute";
```

Comment:
Could you share the docs from the routing? This way we could all have a look at it. Looks good :)